config/interceptor/deployment.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
 spec:
   replicas: 3
   template:
+    metadata:
+      annotations:
+        policy.cilium.io/proxy-visibility: <Ingress/8080/TCP/HTTP>,<Egress/8080/UDP/DNS>,<Egress/80/TCP/HTTP>
```

Comment:
this is not needed. We added it by mistake

Reply:
fixed with [interceptor: remove cilium annotation](https://github.com/wso2/choreo-keda-http-addon-fork/pull/29/commits/3631de38afc22bc564f525eca5aab400268797ac)